templates/src/app/auth/index.js

```diff
@@ -0,0 +1,11 @@
+var { Router } = require("express");
+
+var { authMiddleware } = require("../../middleware/auth");
+
+var router = Router()
+
+router.get("/auth-data", authMiddleware, (req, res) => {
```

Comment:
This is just for an example? Let's stick to camelCase instead of kebab-case for routes. Also maybe this should be something like `/auth/whoami` or `/auth/userInfo` ?